src/ui/views/wallet.rs

```diff
@@ -463,7 +463,6 @@ impl ScreenController for WalletScreenController {
                 app_state_update: AppStateUpdate::ClearedLoadedIdentity,
             }) => {
                 self.identity_info = Info::new_fixed("");
-                self.identity_loaded = false;
```

Comment:
if it's cleared, doesn't it mean identity is not loaded?

Reply:
Same thing, it’s a duplicate line